codemods/babel-plugin-codemod-optional-catch-binding/package.json

```diff
@@ -8,6 +8,9 @@
   "keywords": [
     "@babel/plugin"
   ],
+  "scripts": {
```

Comment:
as this is a published package, i don't think it is a good practice to escape directory boundary, can this be refactored to use a module or something?

Reply:
Hmm, tough one. I agree that escaping boundaries is kind of unfortunate, but making a whole npm package just to hold this script also seems pretty questionable. I was mostly not worrying about it because it's not something people should ever call directly anyway. Did you have something specific in mind?

Reply:
@Andarist I don't know what to say on this. Thoughts?

Reply:
@loganfsmyth Sorry, I've missed the mention here. Imho if it's shareable piece of code between packages it could be put into separate package (even `private` lerna-managed one). That way it should be also easier to discover it and possibly reuse in other packages